tests/phpunit/Integration/MediaWiki/MediaWikiIntegrationForRegisteredHookTest.php

```diff
@@ -61,6 +61,7 @@ protected function tearDown() : void {
 		$this->applicationFactory->clear();
 		$this->mwHooksHandler->restoreListedHooks();
 
+		$this->title = Title::newFromText( __METHOD__ );
```

Comment:
why do we first create a page and then delete it in tearDown()?. Do we need the two lines at all? I'd expect that clean-up is done by MW's test framework now.